modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -1,5 +1,29 @@
 //Z-stream Release Notes by Version
 
+[id="cluster-logging-release-notes-5-3-5"]
+= OpenShift Logging 5.3.5
+This release includes link:https://access.redhat.com/errata/RHSA-2022:0721[RHSA-2022:0721 OpenShift Logging Bug Fix Release 5.3.5]
+
+[id="openshift-logging-5-3-5-bug-fixes"]
+== Bug fixes
+* Before this update, if you removed OpenShift Logging from {product-title}, the web console continued displaying a link to the *Logging* page. With this update, removing or uninstalling OpenShift Logging also removes that link. (link:https://issues.redhat.com/browse/LOG-2182[LOG-2182])
```

Comment:
Should `OpenShift Logging` be replaced with an attribute?

Reply:
No, this is for 5.3, attributes were added for the name change that goes with 5.4